tables/afr-za-g1.ctb

```diff
@@ -49,26 +50,26 @@ include en-ueb-g1.ctb
 #sign \x0308 56 	COMBINING DIAERESIS
 #sign \x0301 4	COMBINING ACUTE ACCENT
 
-uplow	Áá	4-1
-uplow	Àà	45-16-1
+#uplow	Áá	4-1
```

Comment:
Normally I would have said remove the commented lines, but Greg explained this. A comment should be added that these are the old definitions. It's useful to keep them if we ever want to create a separate table for them.